lib/clearance/user.rb

```diff
@@ -143,7 +143,7 @@ module Validations
         validates :email,
           email: { strict_mode: true },
           presence: true,
-          uniqueness: { allow_blank: true },
+          uniqueness: { allow_blank: true, case_sensitive: false },
```

Comment:
@unixmonkey @mjankowski I believe this was supposed to be `case_sensitive: true`.

This actually changes the logic quite a bit, since it builds the queries on `LOWER(email)` instead of `email`. This change broke our app because we didn't have an index on `lower(email)`, so our queries started timing out 😞 

This was originally changed in https://github.com/thoughtbot/clearance/commit/56f0f0489a138d64794136e133c5848183c576d5.

Also, the deprecation warning said we should set this to `true` if want to keep the behavior:

> To **continue** case sensitive comparison
> on the :email attribute in User model, pass `case_sensitive: true`
> option explicitly to the uniqueness validator.

This can make a big difference in performance and requires everyone to change their indexes, so it's not really a backwards compatible change. I'd suggest updating to `case_sensitive: true` and doing a bugfix release.